Formula/openshift-cli.rb

```diff
@@ -52,6 +52,14 @@ def install
     end
   end
 
+  def caveats
+    <<~EOS
+      The version reported by the "oc version --client" command is known to be incorrect.
+      Nevertheless the correct version matching the formula was installed.
+      For more information see: https://github.com/Homebrew/homebrew-core/issues/76968
```

Comment:
Is there no upstream issue we can reference?

Reply:
This is an upstream issue describing this exact problem: https://github.com/openshift/oc/issues/740
I composed a new commit with an updated issue link.